lib/ae_mdw/dex.ex

```diff
@@ -239,12 +297,37 @@ defmodule AeMdw.Dex do
     end
   end
 
+  defp deserialize_debug_contract_swaps_cursor(nil), do: {:ok, nil}
+
+  defp deserialize_debug_contract_swaps_cursor(cursor_bin) do
+    with {:ok, cursor_bin} <- Base.hex_decode32(cursor_bin, padding: false),
+         {{create_txi, create_idx}, txi, log_idx}
+         when is_integer(create_txi) and is_integer(create_idx) and is_integer(txi) and
+                is_integer(log_idx) <-
+           :erlang.binary_to_term(cursor_bin) do
+      {:ok, {{create_txi, create_idx}, txi, log_idx}}
```

Comment:
Maybe using a variable to signify there are no rotation would make it easier to read, as in:
```suggestion
         {{create_txi, create_idx}, txi, log_idx} = cursor
         when is_integer(create_txi) and is_integer(create_idx) and is_integer(txi) and
                is_integer(log_idx) <-
           :erlang.binary_to_term(cursor_bin) do
      {:ok, cursor}
```

Reply:
Done